test/gpu/native/math.chpl

```diff
@@ -260,18 +289,26 @@ foreach i in r do  B[0] =  signbit(r64);   B[1] =  signbit(r64);  check(B,"signb
 foreach i in r do  R[0] =  sin(r32);   R[1] =  sin(r32);  check(R,"sin(r32)");
 foreach i in r do  R[0] =  sin(r64);   R[1] =  sin(r64);  check(R,"sin(r64)");
 foreach i in r do  R[0] =  sinh(r32);   R[1] =  sinh(r32);  check(R,"sinh(r32)");
-foreach i in r do  R[0] =  sinh(r64);   R[1] =  sinh(r64);  check(R,"sinh(r64)");
+if(!excludeForRocm) {
+  foreach i in r do  R[0] =  sinh(r64);   R[1] =  sinh(r64);  check(R,"sinh(r64)");
+}
 
 foreach i in r do  R[0] =  sqrt(r32);   R[1] =  sqrt(r32);  check(R,"sqrt(r32)");
 foreach i in r do  R[0] =  sqrt(r64);   R[1] =  sqrt(r64);  check(R,"sqrt(r64)");
 
 foreach i in r do  R[0] =  tan(r32);   R[1] =  tan(r32);  check(R,"tan(r32)");
-foreach i in r do  R[0] =  tan(r64);   R[1] =  tan(r64);  check(R,"tan(r64)");
+if(!excludeForRocm) {
+  foreach i in r do  R[0] =  tan(r64);   R[1] =  tan(r64);  check(R,"tan(r64)");
+}
```

Comment:
So, there are 64-bit sine and cosine, but no tangent? :(

Reply:
Yeah, this is what really confuses me. If I use `llvm-nm` to look into `ocml.bc` there actually is a `__ocml_tan_f64` symbol but if I use a 64-bit `tan` in `math.chpl` in I'll get `lld: error: undefined hidden symbol: tanf`.

I notice that in `opencl.bc` there is a `_Z3tanf` function but alas linking to that doesn't help either (neither does adding `lopencl`.

I kind of suspect something is resolving to the wrong version of "tanf" somewhere along the line (like resolving to the standard C library version instead of the GPU device version). But I haven't dug deeply into it yet.

Reply:
Perhaps things are getting added gradually, here's a super-helpful link that shows that maybe sine and cosine are also relatively new: https://github.com/RadeonOpenCompute/ROCm-Device-Libs/issues/75